src/main/scala/uk/gov/hmrc/services/validation/models/HelperCaseClasses.scala

```diff
@@ -14,26 +14,23 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.helpers
+package uk.gov.hmrc.services.validation.models
 
-import java.io.{File, FileReader}
 
-import au.com.bytecode.opencsv.CSVReader
-import com.typesafe.config.{Config, ConfigFactory}
-import scala.collection.JavaConverters._
+case class Row(rowNum: Int, cells: Seq[Cell]) {
+  require(cells.forall(_.row == rowNum))
 
-object FileHelper {
+  val cellsByColumn: Map[String, Cell] = cells.map { c => c.column -> c }.toMap
+}
 
-  def readInputCsv(file: File): List[List[String]] = {
-    val reader = new CSVReader(new FileReader(file))
+case class Cell(column: String, row: Int, value: String)
 
-    def readLines(reader: CSVReader): List[List[String]] = {
-      reader.readAll.asScala.toList.map(_.toList)
-    }
-    readLines(reader)
-  }
+case class MissingCell(column: String, row: Int) {
```

Comment:
Isn't this case class redundant? The only time it's used, it just converts into a 'Cell' so you could skip the middle man.

Reply:
it is redundant - we'd left it in for increased readability (to make it obvious that the places where it's called handle a cell that hasn't been provided). happy to remove it if you think it serves no purpose